fastlane/lanes/build.rb

```diff
@@ -166,7 +166,10 @@
       export_options: { **COMMON_EXPORT_OPTIONS, method: 'app-store' }
     )
 
-    upload_build_to_testflight(whats_new_path: WORDPRESS_RELEASE_NOTES_PATH)
+    upload_build_to_testflight(
+      whats_new_path: WORDPRESS_RELEASE_NOTES_PATH,
+      distribution_groups: ['Internal a8c Testers', 'Public Beta Testers']
```

Comment:
I wish ASC had ids for these. I mean, the groups names don't need to be secrets, but it somehow feels out of place to use them. 🤷‍♂️

Reply:
Yeah, agreed. If it makes you more at ease, you could consider declaring them at constants at the top of your Fastfile, so you could use `[INTERNAL_TESTERS, PUBLIC_BETA_TESTERS]` here and make them look a bit more like IDs (in a way) than human-readable strings… but eh 🤷

Reply:
I actually started to define them in that way, but I stopped for two reason.

- The first is a bit of a strawman. Ruby would have required freezing the array, but there have been instances in which Fastlane (or CocoaPods? 🤔 ) would crash because it internally tried to modified the (frozen) input. I wanted to avoid having to deal with that possibility, however remote
- The second is that these are used only once and I don't expect them to be reused, so I thought I might as well save the typing and use literals.